### PR TITLE
Add typings for react-twitter-auth

### PR DESCRIPTION
**Pull request by PaulFasola**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If adding a new definition:
- [x] The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/react-twitter-auth/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
+// Type definitions for react-twitter-auth 0.10
```
Comment (plantain-00) [inline types/react-twitter-auth/index.d.ts]: I think it's 0.0 rather than 0.10

**Review by plantain-00 — CHANGES_REQUESTED** [review]

File: types/react-twitter-auth/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+// Project: https://github.com/GenFirst/react-twitter-auth
+// Definitions by: Paul Fasola <https://github.com/paulfasola>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.6
+
+interface TwitterLoginProp {
```
Comment (DanielRosenwasser) [inline types/react-twitter-auth/index.d.ts]: TwitterLoginProp**s** (notice the trailing `s`)

File: types/react-twitter-auth/index.d.ts (hunk 1 above)
Comment (DanielRosenwasser) [reply types/react-twitter-auth/index.d.ts]: Agreed, please change this to `0.0`

**Review by DanielRosenwasser — CHANGES_REQUESTED** [review]
Looks good but needs two minor fixes.

**Review by plantain-00 — APPROVED** [review]

**Review by DanielRosenwasser — APPROVED** [review]

**Comment by DanielRosenwasser** [comment]
Thanks!

**Pull request merged**